tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/telemetryMetrics/BulkheadTelemetryTest.java

```diff
@@ -206,24 +203,24 @@ public void bulkheadMetricHistogramTest() throws InterruptedException, Execution
         assertThat("histogram count", executionTimesCount, is(2L)); // Rejected executions
                                                                     // not recorded in
 
-        Collection<HistogramPointData> executionTimesPoints = m.getBulkheadRunningDuration().getHistogramPoints();
-        double time = executionTimesPoints.stream()
-                .mapToDouble(points -> points.getSum())
-                .sum();
+        Optional<HistogramPointData> executionTimesPoint = m.getBulkheadRunningDuration().getHistogramPoint();
+        double time = executionTimesPoint
+                .map(HistogramPointData::getSum)
+                .orElse(0.0);
 
-        long count = executionTimesPoints.stream()
-                .mapToLong(points -> points.getCount())
-                .sum();
+        long count = executionTimesPoint
+                .map(HistogramPointData::getCount)
+                .orElse(0L);
 
-        assertThat("mean", Math.round(time / count), approxMillis(1000)); // histogram
+        assertThat("mean", time / count, approxMillisFromSeconds(1000)); // histogram
```

Comment:
This could cause a divide by zero exception if `ft.bulkhead.runningDuration` isn't exported. Not the biggest issue since that's a fail, and it didn't cause problems in the old code, but since we have an Optional anyway we could assert isPresent for a nicer error message.

Reply:
Good point, we should stop the test if there's no data for the metric.

However, I think we still need to assert `count > 0` before computing the mean, since the metric could be present but still have no data in it.

Reply:
Hmm, in fact we're already asserting `count == 2` above but just getting it in a different way... let's tidy this up.

Reply:
Pushed a commit to tidy up this method.